packages/office-ui-fabric-react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -247,11 +269,23 @@ export class CalloutContentBase extends BaseComponent<ICalloutProps, ICalloutSta
   protected _setInitialFocus = (): void => {
     if (this.props.setInitialFocus && !this._didSetInitialFocus && this.state.positions && this._calloutElement.value) {
       this._didSetInitialFocus = true;
-      focusFirstChild(this._calloutElement.value);
+      this._async.requestAnimationFrame(() => focusFirstChild(this._calloutElement.value!));
```

Comment:
There is a new helper focusAsync, which does this. I wonder if focusFirstChild should just call focusAsync.

Reply:
That would make a lot of sense, especially since focus triggers reflows.